### ISSUE TITLE
./configure should check for rst2html

### ISSUE THREAD

**Issue by angband-import** [angband/angband#3870]
**Reported by aerdan on 24 Dec 2013 19:51 UTC**
```./configure``` should check for rst2html and disable building of HTML documentation if not present. Currently, it just stops installation.

**Comment by angband-import** [comment]
**Modified by takkaria on 6 Jan 2014 15:37 UTC**

**Comment by angband-import** [comment]
**Comment by pnd10 on 26 Oct 2014 22:27 UTC**
Following chunk of configure.ac should do most of it:
```
AC_ARG_VAR([RST2HTML], [command for converting reStructuredText to HTML])
AC_CHECK_PROGS([RST2HTML], [rst2html.py rst2html], [NOTFOUND])
AM_CONDITIONAL([HAVE_RST2HTML], [test NOTFOUND != "$RST2HTML"])

if test NOTFOUND = "$RST2HTML"
then
  AC_MSG_NOTICE([rst2html not found; HTML version of manual will not be built])
fi
```

**Comment by angband-import** [comment]
**Modified by magnate on 26 Oct 2014 22:47 UTC**

**Comment by angband-import** [comment]
**Comment by pnd10 on 31 Oct 2014 19:06 UTC**
Given that this only occurs on "make manual", I don't think that configure should print a warning if it's not there. Maybe a warning in the Makefile, if HAVE_RST2HTML is not set, and you try "make manual"?

**Comment by angband-import** [comment]
**Comment by aerdan on 31 Oct 2014 20:05 UTC**
Yes, but `make manual` also produces a PDF version.

**Comment by angband-import** [comment]
**Comment by magnate on 3 Nov 2014 21:18 UTC**
Fixed in 360a6b8 but see also #1831.
